packages/venia-concept/src/components/Pagination/pagination.js

```diff
@@ -117,34 +97,24 @@ class Pagination extends Component {
                 <NavButton
                     name={navButtons.lastPage.name}
                     active={isActiveRight}
-                    onClick={() => this.setPage(rightSkip)}
+                    onClick={() => setPage(rightSkip)}
```

Comment:
Inline callbacks are recreated and reattached on every render. Can we avoid these?

Reply:
Good catch!

Reply:
Gotta be a lint rule somewhere...